mne_bids/stats.py

```diff
@@ -0,0 +1,110 @@
+"""Some functions to extract stats from a BIDS dataset."""
+
+# Authors: Alex Gramfort <alexandre.gramfort@inria.fr>
+#
+# License: BSD (3-clause)
+
+
+from mne_bids import BIDSPath, get_datatypes
+from mne_bids.config import EPHY_ALLOWED_DATATYPES
+
+
+def count_events(root_or_path, datatype='auto'):
```

Comment:
this feels confusing. Why not just `bids_path` instead of `root_or_path`? any use cases not covered by it?

Reply:
I considered it and i find it easier to do

```
from mne_bids.stats import count_events
count_events('/path/...)
```

I consider it also the default usage. So let's not make the default more verbose
of genericity.